File: app/src/main/java/ru/stwtforever/fast/api/model/VKLongPollServer.java
package ru.stwtforever.fast.api.model;
import java.io.*;
import org.json.*;

public class VKLongPollServer extends VKModel implements Serializable {
    
    public String key;
    public String server;
    public long ts;

    public VKLongPollServer(JSONObject source) {
        this.key = source.optString("key");
        this.server = source.optString("server").replace("\\", "");
        this.ts = source.optLong("ts");
    }
}

File: app/src/main/java/ru/melodin/fast/api/model/VKLongPollServer.kt
package ru.melodin.fast.api.model

import org.json.JSONObject

import java.io.Serializable

class VKLongPollServer(source: JSONObject) : VKModel(), Serializable {

    var key: String = source.optString("key")
    var server: String = source.optString("server").replace("\\", "")
    var ts: Long = source.optLong("ts")

    companion object {
        private const val serialVersionUID = 1L
    }
}
